test/general-tests/postman/check.ts

```diff
@@ -0,0 +1,11 @@
+import { exec } from "../../utils/utils";
+import { spawn } from "child_process";
+import { hardhatStarknetCompile, hardhatStarknetTest } from "../../utils/cli-functions";
+
+hardhatStarknetCompile(["contracts/l1l2.cairo"]);
+
+spawn("npx", ["hardhat", "node"], { detached: true });
+exec("sleep 1");
+
+hardhatStarknetTest("--network localhost test/postman.test.ts".split(" "));
+exec("kill -9 $(lsof -t -i:8545)");
```

Comment:
Here we are relying on the fact that the default port is 8545. We could instead provide the port explicitly as an argument of `npx hardhat node` (can be 8545; can be extracted to a var in this file)

Reply:
https://github.com/Shard-Labs/starknet-hardhat-plugin/pull/251/commits/fb0c77a8948621b6eb2c60c6a5b546d431c2a850